Woof/WoofTests/Model/LocalStorage/LocalStorageLoadTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+
+final class LocalStorageLoadTests: XCTestCase {
+    private var localStorage: LocalStorage!
+    private let defaultValueToSave = "Hello, World!".data(using: .utf8)!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        if let bundleIdentifier = Bundle.main.bundleIdentifier {
+            storage.localStorage?.removePersistentDomain(forName: bundleIdentifier)
+        }
+    }
+
+    override func tearDown() {
+        localStorage = nil
+        super.tearDown()
+    }
+
+    func testLoadMethodExistsInAPI() {
+        _ = localStorage.loadValue(for: "")
+    }
+
+    func testLoadValueForExistingKey() {
```

Comment:
The name of test doesn't reflect what happens.
Please review the rest of code.

Reply:
fixed